src/build/env.py

```diff
@@ -123,7 +123,9 @@ def install(self, requirements: Collection[str]) -> None:
         if not requirements:
             return
 
-        self.log(f'Installing packages in isolated environment... ({", ".join(sorted(requirements))})')
+        self.log(
+            f'Installing packages in isolated environment... ({", ".join(_quote_text(req) for req in sorted(requirements))})'
```

Comment:
I think there's an `shlex.quote()` helper doing this already. Try using it.

Also, nested things within f-strings are hard to perceive sanely. This calls for a helper variable, I believe.

Reply:
Let's just have a sub-list, it'll be easier to scan:

```
* Installing packages in isolated environment:
  - setuptools
  - wheel
```